src/AutoMerge/toml.jl

```diff
@@ -0,0 +1,18 @@
+function maybe_parse_toml(directory::AbstractString, file::AbstractString)
+    allowed_filenames = (
+        "Compat.toml",
+        "WeakCompat.toml",
+        "Deps.toml",
+        "WeakDeps.toml",
+    )
+    if !(file in allowed_filenames)
+        msg = "Filename is not in the allowed list: $(file)"
+        throw(ErrorException(msg))
+    end
+    full_path = joinpath(directory, file)
+    if !ispath(full_path)
+        @warn "TOML file does not exist; returning an empty dict" directory file full_path
```

Comment:
And why is this a warning? Getting an empty dict when the file does not exist is the point of the function and why you call it in the first place. So why should it warn?

Reply:
The warning is now removed.